asistente_ladm_col/gui/reports/reports.py

```diff
@@ -258,6 +262,7 @@ def generate_report(self, db, report_type):
                     functools.partial(self.stdout_ready, proc=proc))
 
                 parcel_number = self.ladm_data.get_parcels_related_to_plots(db, [plot_id], db.names.LC_PARCEL_T_PARCEL_NUMBER_F) or ['']
+                self.app.core.get_layer(db, db.names.LC_PLOT_T, load=True)  # previous function changed the selected layer, plots layer are selected again
```

Comment:
This is just to **select** the plot? If so, this is not the best function to use.

Reply:
you're right it's not the best way, It is only necessary to leave the plot layer as active layer in the TOC. We should use something like that `self.app.gui.activate_layer(plot_layer)`. I did the test and it worked correctly

Reply:
Good